PYIC-2739: Added call to /journey/check-existing-identity through JSON request

chris-dco opened the pull request:
## Proposed changes

### What changed

Reconfigured /journey/check-existing-identity API Gateway endpoint to call the check-existing-identity Lambda using a plane JSON request rather than an AWS Proxy request.

### Why did it change

So that the check-existing-identity Lambda can be migrated to be called by the Process Journey Step State Machine.

### Issue tracking

- [PYIC-2739](https://govukverify.atlassian.net/browse/PYIC-2739)

## Checklists

### Environment variables or secrets
- [X] No environment variables or secrets were added or changed

### Other considerations

None

[PYIC-2739]: https://govukverify.atlassian.net/browse/PYIC-2739?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

therealslimsladey committed:
PYIC-2709: Make ConfigurationVariable paths relative

therealslimsladey committed:
PYIC-2710: Add featureSet overrides in scalar methods and tests thereof

therealslimsladey committed:
PYIC-2710: Add featureSet overrides in scalar methods

therealslimsladey committed:
PYIC-2708: Replace CREDENTIAL_ISSUERS_CONFIG_PARAM_PREFIX with ENVIRONMENT equivalent

therealslimsladey committed:
PYIC-2777: Add featureSet getter/setter

kerrr committed:
BAU: Remove logging of PII

We should *never* log PII.

chris-dco committed:
PYIC-2739: Added call to /journey/check-existing-identity through JSON request

chris-dco committed:
PYIC-2739: Removed debugging information

chris-dco committed:
PYIC-2739: Fixed spotless issues

blakeyp commented on openAPI/core-back-internal.yaml:
```diff
@@ -271,7 +271,24 @@ paths:
-        type: "aws_proxy"
+        type: "aws"
+        requestTemplates:
+          application/x-www-form-urlencoded:
```
Wouldn't this be `application/json` ?

chris-dco replied:
Apparently not. When I checked it's called with application/x-www-form-urlencoded.

vam-gds committed:
PYIC-2781 Updated Deployment pipeline

This should allow both development accounts to work side-by-side
	modified:   deploy/template.yaml

therealslimsladey committed:
PYIC-2708: Add more config service tests

chris-dco committed:
PYIC-2742 Update select-cri to support both API Gateway and JSON requests

chris-dco committed:
PYIC-2739: Updated merge conflict of JourneyRequest.java
